.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,57 @@
+name: Bug Report
+description:  Create a report to help us improve
+labels: ["Type: Bug", "Needs: Triage", "Needs: Lead"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **IMPORTANT: Before posting, remove sensitive data, such as passwords, secret keys, session cookies, etc.**
+
+        Thanks for taking the time to fill out this bug report! 👍
+  - type: textarea
+    attributes:
+      label: Problem
+      description: A short description of the issue you're facing.
+      value: |
+        #### Describe the bug
```

Comment:
Let's try without the extra heading here; it makes the hover cards over issues fill up with the template. Hopefully folks will get they can add data!

```suggestion
        
```

eg this is kind of wasted space
![image](https://github.com/internetarchive/openlibrary/assets/6251786/6ceccc95-9501-4bbf-961c-404933c68266)